Gems/Atom/Feature/Common/Assets/Materials/Types/StandardMultilayerPBR.materialtype

```diff
@@ -504,9 +513,17 @@
             "irradiance": [
                 // Note: this property group is used in the DiffuseGlobalIllumination pass, it is not read by the StandardPBR shader
                 {
-                    "name": "color",
-                    "displayName": "Color",
-                    "description": "Color is displayed as sRGB but the values are stored as linear color.",
+                    "name": "irradianceColorSource",
+                    "displayName": "Irradiance Color Source",
+                    "description": "Where to get the irradiance color from? This color can be automatically derived from the diffuse base color (average texture color if textured), or the manually provided color below.",
+                    "type": "Enum",
+                    "enumValues": [ "BaseColor", "Manual"],
+                    "defaultValue": "BaseColor"
```

Comment:
Hm, I don't think we talked about this before, I just realized that using the BaseColor or BaseColorTint modes won't work for StandardMultilayerPbr because it doesn't have a baseColor property group, it has layer1_baseColor, layer2_baseColor, and layer3_baseColor. And you can't know which one is actually active without evaluating the surface. I think we can't support automatic irradiance color for this material type. We should just revert back to how it was before, with a single irradiance.color property.